app/assets/javascripts/rad_common/autocomplete-rails.js

```diff
@@ -126,16 +126,30 @@
           if(jQuery(e).data('excluded-ids')) {
               params['excluded_ids'] = jQuery(e).data('excluded-ids');
           }
+          const allowNewModel = jQuery(e).attr('data-allow-new-model');
```

Comment:
@calebtocco I think you can do
`jQuery(e).data('allow-new-model'`

which will automatically cast it to truthy value so no need for !!

Reply:
cool ill give that a try